src/vendor/wordpress/_wordpress.scss

```diff
@@ -0,0 +1,52 @@
+@use "../../mixins/button";
+@use "../../design-tokens/colors.yml";
+
+/**
+ * Gutenberg block: Button
+ * Applies our pattern library styles to buttons generated via Gutenberg blocks.
+ */
+
+.wp-block-button {
+  @include button.default;
+}
+
+.wp-block-button.is-style-fill {
```

Comment:
Can you nest `&.is-style-fill` and `&.is-style-outline` inside `.wp-block-button`?

Reply:
Good suggestion. Thanks @spaceninja! 

> Overall this seems reasonable to me. I'll defer to @tylersticka, though.

During my 1:1 with Tyler earlier this morning, he mentioned he had some general high-level feedback about the overall approach here, so I'm going to wait and see how his feedback affects this suggestion before pushing any updates.